packages/react-core/src/evm/utils/cache-keys.ts

```diff
@@ -182,10 +182,12 @@ export const cacheKeys = {
           ),
         totalCirculatingSupply: (
           contractAddress: RequiredParam<ContractAddress>,
+          tokenId?: RequiredParam<BigNumberish>,
         ) =>
           createContractCacheKey(contractAddress, [
             "query",
             "totalCirculatingSupply",
+            tokenId ?? "erc721",
```

Comment:
```suggestion
            tokenId ?? 0,
```

tokenId will be a BigNumberIsh so I would simply go with `0` here for erc721 case (erc721 can only ever be one token so the "tokenId" would be `0` if it had one, seems reasonable)

also just a note for additional learning, no need to make this change here:
- the _correct_ hierarchy for the query key (cache key) here would be `contract -> query -> tokenId -> totalCirculatingSupply
- in this case it's OK to leave it as is as we actually never invalidate cache for  individual tokens anywhere today, but in general the principle should be "least significant -> "most significant"

Reply:
🙏 🙏